cmd/ci-operator/main.go

```diff
@@ -216,6 +213,7 @@ func bindOptions(flag *flag.FlagSet) *options {
 	flag.StringVar(&opt.namespace, "namespace", "", "Namespace to create builds into, defaults to build_id from JOB_SPEC. If the string '{id}' is in this value it will be replaced with the build input hash.")
 	flag.StringVar(&opt.baseNamespace, "base-namespace", "stable", "Namespace to read builds from, defaults to stable.")
 	flag.DurationVar(&opt.idleCleanupDuration, "delete-when-idle", opt.idleCleanupDuration, "If no pod is running for longer than this interval, delete the namespace. Set to zero to retain the contents.")
+	flag.DurationVar(&opt.cleanupDuration, "delete-after", opt.cleanupDuration, "If namespace exists for longer than this interval, delete the namespace. Set to zero to retain the contents.")
```

Comment:
Not entirely true now that this depends on a separate controller being present.

Reply:
What would you suggest>

Reply:
I'd add "(requires openshift/ci-ns-ttl-controller)" or something.

You probably won't care if you're invoking on the command line or your own
cluster, but it's worth documenting.

Reply:
done